tests/baselines/reference/outModuleConcatES6.js

```diff
@@ -0,0 +1,22 @@
+//// [tests/cases/compiler/outModuleConcatES6.ts] ////
+
+//// [a.ts]
+export class A { }
+
+//// [b.ts]
+import {A} from "./ref/a";
+export class B extends A { }
+
+
+
+!!!! File all.d.ts missing from original emit, but present in noCheck emit

```

Comment:
In general, how are we ending up with these messages? Are they all cases where we don't emit due to errors?

Reply:
Yea, a checker error blocks the .d.ts emit without `noCheck` on.